mne_bids/read.py

```diff
@@ -249,6 +250,52 @@ def read_raw_bids(bids_fname, bids_root, verbose=True):
     return raw
 
 
+def get_matched_empty_room(bids_fname, bids_root):
+    """Get matching empty room file.
+
+    Parameters
+    ----------
+    bids_fname : str
+        The filename for which to find the matching empty room file.
+    bids_root : str
+        Path to the BIDS root folder.
+
+    Returns
+    -------
+    er_fname : str | None.
+        The filename corresponding to the empty room.
+        Returns None if no file found.
+    """
+    from mne_bids.write import make_bids_folders
```

Comment:
why relative import?

Reply:
because this file is imported in write.py -- I thought it could lead to circular import issue.

Reply:
> I thought it could lead to circular import issue.

Have you tried it? Or is it just a suspicion?

Reply:
if we end up with circular imports with such as small project it's a design pb we should fix early

Reply:
Let's fix this in another issue. I would like to keep the scope of this PR constrained. It already depends on many other PRs in MNE-Python. Opened an issue to deal with it. See: https://github.com/mne-tools/mne-bids/issues/299

Reply:
okay fair enough. I fixed this in the last commit